en/advanced_features/precland.md

```diff
@@ -1,114 +1,135 @@
 # Precision Landing
 
-PX4 supports precision landing for multicopters with the [IRLock](https://irlock.com/products/ir-lock-sensor-precision-landing-kit) sensor, an IRLock beacon, and a downward facing range sensor.
-This enables landing with a precision of roughly 10 cm, as opposed to the precision given by GPS, which may be several meters.
+PX4 supports precision landing for *multicopters* using the [IR-LOCK Sensor](https://irlock.com/products/ir-lock-sensor-precision-landing-kit), an IR beacon (e.g. [IR-LOCK MarkOne](https://irlock.com/collections/markone)), and a downward facing range sensor.
```

Comment:
>IR beacon (e.g. IR-LOCK MarkOne)

This might be too generic. The IRLock camera only detects the IRLock "Mark one" beacon (https://irlock.com/products/markone-beacon-v2-0) with the newest firmware.

Reply:
Thanks! I've added that as a job to fix in #100